gce/burst-training/census-analysis.py

```diff
@@ -0,0 +1,126 @@
+import argparse
+import joblib
```

Comment:
`joblib` could be imported also as `sklearn.external.joblib`, which *might* be better.

Reply:
Did not know about that. Although it's cool, I feel this is more standard and prefer this setup.

Reply:
There could be version conflicts - the version of joblib included in sklearn (probably a few versions behind joblib itself) might be better tested with sklearn's other components.

Reply:
This is not the case at the moment and the requirements file forces installation of the same versions I am using locally (which do not cause conflict).

Do you feel strongly about this? I prefer the tidiness of `import joblib` and do not mind the redundancy at all. This reduces the number of changes a user would have to make if they were not using `sklearn`.

Reply:
I don't feel strongly about this.

Reply:
Cool.

Incidentally, today we found a case where you may want to import from sklearn.joblib (`joblib` itself may not be available in Dataflow's Python environment.)